cypress/support/helpers/getPaths.js

```diff
@@ -0,0 +1,14 @@
+import config from '../config/services';
+import getAppEnv from './getAppEnv';
+
+export default (service, pageType) => {
```

Comment:
I always try to avoid using `let` if i can since it's essentially mutable state which can make code harder to reason about. What about

```js
export default (service, pageType) => {
  const { paths = [] } = config[service].pageTypes[pageType];

  return paths[getAppEnv()] || [];
};
```